### DISCUSSION TITLE
Where to download the dataset？

### DISCUSSION THREAD

**Discussion by Rainbowman0** [MkuuWaUjinga/leopart#2] [Q&A] [answered]
Hi, I would like to know where to download the VOC Pascal dataset in this project. The readme says "The data is encapsulated in lightning data modules. Please download the data and organize as detailed in the next subsections." The folder structure of the VOC Pascal dataset I downloaded from the official website is different from yours. Specifically, there is no "SegmentationClassAug" folder, which puzzles me a bit.

Where can I get the VOC Pascal dataset in this project? Looking forward to your reply. :)

**Comment by MkuuWaUjinga** [comment, marked as answer]
Hi, sorry for the late reply! I added the zipped archive to the section you mentioned (https://github.com/MkuuWaUjinga/leopart#voc-pascal). "SegmentationClassAug" refers to the extended PASCAL training set used by prior works such as MaskContrast, or Hierarchical Grouping.
Let me know in case you have any other questions :)
